app/models/plant_population.rb

```diff
@@ -25,6 +25,7 @@ class PlantPopulation < ActiveRecord::Base
 
   validates :name,
             presence: true,
+            uniqueness: true,
             allow_blank: true
```

Comment:
Does uniqueness and allow_blank make any sense together? This just means there could be a single blank PP.name, right?